problems/getCountriesSortedByPopulation.js

```diff
@@ -7,8 +7,9 @@
 * 
 */
 
-function getCountriesSortedByPopulation() {
+function getCountriesSortedByPopulation(arr) {
+      return arr.sort((a, b) => {
+       b.population - a.population})
```

Comment:
very close! missing a return for the callback function, this will sort them from largest to smallest. 
Lastly you'll need to chain a call to map to return the array with only the properties that you desire.